examples/pom.xml

```diff
@@ -12,8 +12,8 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <!-- change to the version you want to use. -->
     <jetty.version>9.4.17.v20190418</jetty.version>
-    <opencensus.version>0.27.0</opencensus.version><!-- LATEST_OPENCENSUS_RELEASE_VERSION -->
-    <grpc.version>1.27.2</grpc.version><!-- CURRENT_GRPC_VERSION -->
+    <opencensus.version>0.27.1</opencensus.version> <!-- LATEST_OPENCENSUS_RELEASE_VERSION -->
+    <grpc.version>1.32.2</grpc.version> <!-- CURRENT_GRPC_VERSION -->
```

Comment:
This breaks the build for **maven only**:

```
[ERROR] Failed to execute goal on project opencensus-examples: Could not
resolve dependencies for project
io.opencensus:opencensus-examples:jar:0.28.0-SNAPSHOT: Failed to collect
dependencies for io.opencensus:opencensus-examples:jar:0.28.0-SNAPSHOT: Could
not resolve version conflict among
[io.opencensus:opencensus-exporter-stats-stackdriver:jar:0.27.1 ->
io.grpc:grpc-auth:jar:1.27.2 -> io.grpc:grpc-api:jar:[1.27.2,1.27.2],
io.grpc:grpc-core:jar:1.32.2 -> io.grpc:grpc-api:jar:[1.32.2,1.32.2],
io.grpc:grpc-protobuf:jar:1.32.2 -> io.grpc:grpc-api:jar:1.32.2,
io.grpc:grpc-protobuf:jar:1.32.2 -> io.grpc:grpc-protobuf-lite:jar:1.32.2 ->
io.grpc:grpc-api:jar:1.32.2, io.grpc:grpc-stub:jar:1.32.2 ->
io.grpc:grpc-api:jar:1.32.2, io.grpc:grpc-census:jar:1.32.2 ->
io.grpc:grpc-api:jar:[1.32.2,1.32.2]] -> [Help 1]
```

I believe it works fine for gradle because the stackdriver exporter's `build.gradle` has this:
https://github.com/census-instrumentation/opencensus-java/blob/8564d8146a315370ccd1c7ccf3d98adc9c128098/exporters/trace/stackdriver/build.gradle#L34-L34

I'm not too familiar with gradle, any ideas for a fix?

Reply:
Nvm, this seems to fix it 
```xml
<exclusion>
  <groupId>io.grpc</groupId>
  <artifactId>grpc-api</artifactId>
</exclusion>
```